test/workbox-webpack-plugin/node/generate-sw.js

```diff
@@ -274,6 +274,120 @@ describe(`[workbox-webpack-plugin] GenerateSW (End to End)`, function() {
       });
     });
 
+    it(`should support setting importWorkboxFrom to 'local', respecting output.publicPath`, function(done) {
+      const FILE_MANIFEST_NAME = 'precache-manifest.ded0a9e68272d64590dbeebac7c52fd7.js';
+      const outputDir = tempy.directory();
+      const publicPath = '/testing/';
+      const config = {
+        entry: {
+          entry1: path.join(SRC_DIR, WEBPACK_ENTRY_FILENAME),
+          entry2: path.join(SRC_DIR, WEBPACK_ENTRY_FILENAME),
+        },
+        output: {
+          filename: '[name]-[chunkhash].js',
+          path: outputDir,
+          publicPath,
+        },
+        plugins: [
+          new GenerateSW({importWorkboxFrom: 'local'}),
+        ],
+      };
+
+      const compiler = webpack(config);
+      compiler.run(async (webpackError) => {
+        if (webpackError) {
+          return done(webpackError);
+        }
+
+        const swFile = path.join(outputDir, 'service-worker.js');
+        try {
+          // Validate the copied library files.
+          const libraryFiles = glob.sync(`${WORKBOX_DIRECTORY_PREFIX}*/*.js*`,
+            {cwd: outputDir});
+
+          const modulePathPrefix = path.dirname(libraryFiles[0]);
+
+          const basenames = libraryFiles.map((file) => path.basename(file));
+          expect(basenames).to.eql([
```

Comment:
My only passing comment is that it appears lines 197 and 311 execute `expect(basenames).to.eql()` against the same list of dev, prod and map files - should this list of basenames be turned into a variable if they're the same between both tests? If I missed something and they're not exactly 1:1, feel free to ignore.

Reply:
Good point. Changing!